### DISCUSSION TITLE
Passing A Segment Built in R to aw_freeform_table

### DISCUSSION THREAD

**Discussion by davisj95** [benrwoodard/adobeanalyticsr#152] [Q&A] [answered]
I just discovered this package and I'm excited to use it! I see that there is an option to build a segment using ```seg_build()```, but I wanted to know if there were a way to build the segment on-the-fly and pass it into ```aw_freeform_table()```? I see in the documentation for ```aw_freeform_table()``` that there is an option to include a segmentId, but I'm hoping to create segments within R and pass them into this function without necessarily publishing the segment and getting a segmentId from it.

**Comment by benrwoodard** [comment, marked as answer]
Hey Jake! Adding a custom or on-the-fly segment to the aw_freeform_table() is definitely something we are working on. We haven't finalized it yet but hope to have it going by the time we update the package to the next version before the EOY.
